### DISCUSSION TITLE
Session macros not available as a sidebar control in drum velocity keyboard view

### DISCUSSION THREAD

**Discussion by seangoodvibes** [SynthstromAudible/DelugeFirmware#2997] [Ideas]
### Please describe the problem:

Session macros not available as a sidebar control in drum velocity keyboard view

### What is the expected behavior?

Session macros are available in drum velocity keyboard view sidebar

### What hardware did you reproduce it with?

OLED

### What firmware did you reproduce it with?

Beta

**Comment by m-m-adams** [comment]
Are any columns available?

**Comment by m-m-adams** [comment]
This is a fairly large refactor - moving it to discussions since it's definitely a feature request and not a bug
